src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -87,6 +87,7 @@
 #include <Psychrometrics.hh>
 #include <ReportSizingManager.hh>
 #include <ScheduleManager.hh>
+#include <SetPointManager.hh>
```

Comment:
will be calling specific SPM from CBVAV, this may come in handy until ManageSetPoints is moved into the HVAC iteration loop.